puppet/zulip/templates/nginx.conf.template.erb

```diff
@@ -13,6 +13,7 @@ events {
 }
 
 http {
+    ssi on;
```

Comment:
We definitely do not want to enable SSI for everything we serve. This would include user-uploaded content.